subiquity/models/filesystem.py

```diff
@@ -1092,7 +1093,10 @@ def name(self):
         return self.pool
 
     @property
-    def mount(self):
+    def path(self):
+        if self.fs_properties is not None:
+            if not yaml.safe_load(self.fs_properties.get('canmount', 'off')):
```

Comment:
Oh right this is partly about what curtin accepts for this field too isn't it. It looks like curtin accepts booleans in this field as well as 'on' and 'off' strings (because lets face it, noone is going to remember to write `canmount: 'on'` rather than `canmount: on` in their yaml) -- but of course "safe_load(True)" explodes. So I think code interpreting these fields needs to accept: True, False, "on", "off" and "noauto". But no others.

Does that make sense? I'm not sure my brain is working at "making statements about technology I don't really understand" levels today.

Reply:
Done